governance/reviewers-maintainers.md

```diff
@@ -1,40 +1,19 @@
-Unlike other Special Interest Groups (SIGs), the O3DE Documentation and Community SIG maintains two reviewer roles requiring SIG approval: *Documentation reviewer* and *technical reviewer*. Learn how to become either, and file a [nomination issue](https://github.com/o3de/sig-docs-community/issues/new?assignees=&labels=&template=reviewer-maintainer-nomination.md&title=SIG+Reviewer%2FMaintainer+Nomination) to become an official reviewer or maintainer!
+## Documentation and Community reviewer roles
 
-For more on the Reviewer and Maintainer roles, read our [Community Membership](https://github.com/o3de/community/blob/main/community-membership.md) information. 
+The Documentation and Community SIG maintains the following reviewer pools:
 
-----
-<!-- CUT ABOVE THIS LINE BEFORE POSTING -->
-### Nomination Guidelines
+* **Documentation Reviewer** - Responsible for reviewing documentation content to `o3de.org`, or conducting reviews of certain `o3de` in-product documentation elements.
```

Comment:
- Add link to team
- I'm not entirely sure what "in-product documentation elements" means. Perhaps you can give a simple example? 
```suggestion
* [**Documentation Reviewer**](https://github.com/orgs/o3de/teams/docs-reviewers) - Responsible for reviewing documentation content to `o3de.org`, or conducting reviews of certain `o3de` in-product documentation elements.
```